content/whats-next/whats-next-mid-term.json

```diff
@@ -14,16 +14,6 @@
     "description": "Today, we provide one product identifier named `sku`. It can be a bit too restrictive as some of you use other identifiers such as GTIN for example. Let's make it possible for you to choose a different naming!",
```

Comment:
In the last sentence, we say 'to choose a different naming!' is that really what's happening? is it about the naming itself? in my opinion that is making this release seem smaller than it will be. suggestion -- maybe we keep it more ambiguous and say "That's why we're working to create a more flexible product identifier!

Reply:
This is 'old news', I didn't change it and I think we should leave it as is.
About the screenshot, I'll check with Stephane!